rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/filter/TbMsgTypeSwitchNode.java

```diff
@@ -85,6 +85,14 @@ public void onMsg(TbContext ctx, TbMsg msg) {
             relationType = "Alarm Acknowledged";
         } else if (msg.getType().equals(DataConstants.ALARM_CLEAR)) {
             relationType = "Alarm Cleared";
+        } else if (msg.getType().equals(DataConstants.ALARM_ASSIGN)) {
```

Comment:
I don't like the diff between names: 

* ALARM_ASSIGN but not  ALARM_ASSIGNED
* however the relation type is "Alarm Assigned"